app/templates/direct-award/download-search-results.html

```diff
@@ -53,13 +53,26 @@
       </ul>
     </div>
     <div class="marketplace-paragraph">
-      <p>
-        <br>
-        <a href="{{ url_for('direct_award.download_shortlist', framework_framework=framework.framework, project_id=project.id, filetype='odf') }}">Download search results spreadsheet (ODF)</a>
-        <br>
-        <a href="{{ url_for('direct_award.download_shortlist', framework_framework=framework.framework, project_id=project.id, filetype='csv') }}">Download search results spreadsheet (CSV)</a>
-      </p>
       <br>
+      {%
+      with
+      items = [
+          {
+              "title": "Download search results as a spreadsheet",
+              "link": url_for('direct_award.download_shortlist', framework_framework=framework.framework, project_id=project.id, filetype='odf'),
+              "file_type": "ODS",
+              "download": "True"
+          },
+          {
+              "title": "Download search results as comma-seperated values",
```

Comment:
Spelling: sep*a*rated

Reply:
👍

Reply:
:man_facepalming: I get that wrong all the time.